content_scripts/CameraVis.js

```diff
@@ -1314,28 +1322,73 @@ void main() {
             });
         }
 
+        restorePatch(serialization) {
+            const containerMatrix = new THREE.Matrix4().fromArray(serialization.container);
+            const phoneMatrix = new THREE.Matrix4().fromArray(serialization.phone);
+            const textureImage = document.createElement('img');
+            textureImage.src = serialization.texture;
+            const textureDepthImage = document.createElement('img');
+            textureDepthImage.src = serialization.textureDepth;
+
+            const patch = CameraVis.createPatch(
+                containerMatrix,
+                phoneMatrix,
+                textureImage,
+                textureDepthImage
+            );
+            patch.__creationTime = serialization.creationTime;
```

Comment:
Is there a 'flash' animation (in createPatch) for each restored patch? Does it make sense at this point of execution?